packages/trader/src/AppV2/Components/BottomNav/BottomNavItem.tsx

```diff
@@ -0,0 +1,29 @@
+import { CaptionText } from '@deriv-com/quill-ui';
+import classNames from 'classnames';
+import React from 'react';
+
+type BottomNavItemProps = {
+    icon: React.ReactNode;
+    selectedIndex: number;
+    label: React.ReactNode;
+    setSelectedIndex: (index: number) => void;
+    index: number;
+};
+
+const BottomNavItem = ({ icon, selectedIndex, label, index, setSelectedIndex }: BottomNavItemProps) => {
+    const isActive = index === selectedIndex;
+    return (
+        <button
+            className={classNames('bottom-nav-item', isActive ? 'bottom-nav-item--active' : '')}
+            onClick={() => setSelectedIndex(index)}
+        >
+            <span>{icon}</span>
+            {/* //waiting on quill-tokens */}
+            <CaptionText size='sm' color='var(--core-color-opacity-black-600)'>
```

Comment:
Not sure, but maybe we need to use `<Text/>` instead of `<CaptionText/>`? It will help to get rid of color prop
![Screenshot 2024-05-10 at 2 08 09 PM](https://github.com/binary-com/deriv-app/assets/121025168/7a8a0224-e185-491a-96b9-2c42242fa1f6)

Reply:
just above where you've underline it says

mobile/semantic/typography/caption/regular/default. Not sure what to use